### ISSUE TITLE
New Problem: Lasso Regression 

### ISSUE THREAD

**Issue by moe18** [Open-Deep-ML/DML-OpenProblem#30]
Implement Lasso Regression using numpy only

reference: https://scikit-learn.org/stable/modules/linear_model.html#lasso

**Comment by afrenkai** [comment]
Would love to tackle this! Would we want a sparse solution utilizing coordinate descent or a simpler one utilizing gradient descent? In case 1: 
<img width="260" alt="image" src="https://github.com/user-attachments/assets/a0b6891c-0106-4e2d-8599-df434e2f5ec5">
and in case 2: 
<img width="484" alt="image" src="https://github.com/user-attachments/assets/00771030-2d2c-4266-befd-1c8148823305">
I can update the learn.html with whatever you choose is best.

**Comment by Open-Deep-ML** [comment]
you could make two questions one for each method, make it problem 50 and 51, or if you just want to do one do case 2

**Comment by afrenkai** [comment]
Sounds great. Will do.
